src/dfs/dfs_rhs_new.cpp

```diff
@@ -64,9 +64,9 @@ namespace SpherePoisson {
 
         // compute fft using fftw3
         fftw_plan plan;
-        int threads = 8;
+        int nthreads = omp_get_max_threads();
```

Comment:
Does this mean that DFS requires OpenMP?   If so, we should have the CMake system enforce this condition.